src/main/java/com/iota/iri/service/dto/GetNodeInfoResponse.java

```diff
@@ -25,12 +25,14 @@
     private long time;
     private int tips;
     private int transactionsToRequest;
+    private String coordinatorAddress;
 
 	public static AbstractResponse create(String appName, String appVersion, int jreAvailableProcessors, long jreFreeMemory,
 	        String jreVersion, long maxMemory, long totalMemory, Hash latestMilestone, int latestMilestoneIndex,
 	        Hash latestSolidSubtangleMilestone, int latestSolidSubtangleMilestoneIndex, int milestoneStartIndex,
 	        int neighbors, int packetsQueueSize,
-	        long currentTimeMillis, int tips, int numberOfTransactionsToRequest) {
+	        long currentTimeMillis, int tips, int numberOfTransactionsToRequest,
+			String coordinatorAddress) {
```

Comment:
this is the style guide speaking... I will align it back -- this file uses tabs :/